### ISSUE TITLE
"__webpack_require__ is not defined"

### ISSUE THREAD

**Issue by id-pogni** [infielddigital/aem-webpack-example#47]
Browser console logs: `__webpack_require__ is not defined` in development mode. This issue does not occur in production builds.

![screen shot 2018-11-13 at 5 08 17 pm](https://user-images.githubusercontent.com/2172103/48453122-edf90300-e766-11e8-9c88-0c67a4581bb5.png)

Adding a devtool configuration to webpack.config.base.js, like `devtool: 'inline-source-map',` resolves the issue.

**Comment by kevinweber** [comment]
I just download master and tested `npm start` two different versions of Node, and I wasn't able to reproduce the error.

Have you added/updated any dependencies to your setup? Have you modified `.babelrc`?

Can you check if there's a `.babelrc` file somewhere in a parent directory that doesn't look like the one defined here: […/main/webpack.core/.babelrc](/infielddigital/aem-webpack-example/blob/master/ui.apps/src/main/webpack.core/.babelrc)? For example, do you have such a file in […/main/](/infielddigital/aem-webpack-example/blob/master/ui.apps/src/main/) in addition/instead of the one in […/main/webpack.core/](/infielddigital/aem-webpack-example/blob/master/ui.apps/src/main/webpack.core/)?

**Comment by max-ch9i** [comment]
Is this perchance not because of the minifier? I fixed the error by adding
`jsProcessor="[default:none,min:gcc;languageIn=ECMASCRIPT_2015;compilationLevel=whitespace]"` to the clientlib .content.xml.

**Comment by dpogni** [comment]
You're right @maximus8891, this is GCC related. Thank you for the fix suggestion
